README.md

```diff
@@ -34,46 +35,103 @@ snowdrop	    https://snowdrop.github.io/helm
-| spring-boot-example-app        | Chart to be used by the Snowdrop Spring Boot examples | [values.yaml](repository/spring-boot-example-app/values.yaml) |
+| Chart Name                                                             | Description | Source |
+|------------------------------------------------------------------------|-------------| ------ |
+| [spring-boot-example-app](#spring-boot-example-app)                    | Chart to be used to create a Spring Boot application | [repository/spring-boot-example-app](repository/spring-boot-example-app) |
+| [spring-boot-example-rest-http](#spring-boot-example-rest-http)        | Chart to deploy the Snowdrop Spring Boot REST HTTP example | [https://github.com/snowdrop/rest-http-example](https://github.com/snowdrop/rest-http-example) |
+| [spring-boot-example-cache](#spring-boot-example-cache)                | Chart to deploy the Snowdrop Spring Boot Cache example | [https://github.com/snowdrop/cache-example](https://github.com/snowdrop/cache-example) |
+| [spring-boot-example-crud](#spring-boot-example-crud)                  | Chart to deploy the Snowdrop Spring Boot CRUD example | [https://github.com/snowdrop/crud-example](https://github.com/snowdrop/crud-example) |
+| [spring-boot-example-configmap](#spring-boot-example-configmap)        | Chart to deploy the Snowdrop Spring Boot ConfigMap example | [https://github.com/snowdrop/configmap-example](https://github.com/snowdrop/configmap-example) |
+| [spring-boot-example-health-check](#spring-boot-example-health-check)  | Chart to deploy the Snowdrop Spring Boot Health Check example | [https://github.com/snowdrop/health-check-example](https://github.com/snowdrop/health-check-example) |
+| [spring-boot-example-circuit-breaker](#spring-boot-example-circuit-breaker)  | Chart to deploy the Snowdrop Spring Boot Circuit Breaker example | [https://github.com/snowdrop/circuit-breaker-example](https://github.com/snowdrop/circuit-breaker-example) |
+| [spring-boot-example-messaging-queue](#spring-boot-example-messaging-queue)  | Chart to deploy the Snowdrop Spring Boot Messaging Queues example | [https://github.com/snowdrop/messaging-work-queue-example](https://github.com/snowdrop/messaging-work-queue-example) |
 
 ## Usage
 
-To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
+### spring-boot-example-app
+
+This chart deploys and exposes a Spring Boot application using S2i.
+To install it, you need to execute the following command:
+
+```console
+$ helm install my-spring-boot-app snowdrop/spring-boot-example-app --set name=app --set s2i.source.repo=http://github.com/org/repo --set s2i.source.ref=main
+```
+
+When installing the chart, Openshift will trigger a S2i build to clone the GIT repository `http://github.com/org/repo` using the branch `main`, and then will execute Maven to build the application JAR to run on pods.
```

Comment:
`When installing the chart, Openshift will trigger a S2i build` ==> `When you install the chart on OpenShift, a pod is created to build the SB application using as source the GIT repository cloned and the maven tool. The jar file generated will be copied to an image and pushed to a registry.`

Reply:
Changed